src/qt/guiutil.cpp

```diff
@@ -956,4 +956,13 @@ void PolishProgressDialog(QProgressDialog* dialog)
 #endif
 }
 
+int TextWidth(const QFontMetrics& fm, const QString& text)
```

Comment:
1260ecd812e35185898fd555ad3e01d019072bcf

Isn't it better to use a deprecated function instead of implementing workarounds? IMHO deprecated stuff can be removed only when it's 1-to-1. So in this case this change sounds premature.

Reply:
> IMHO deprecated stuff can be removed only when it's 1-to-1.

Unfortunately, Qt's "support version window" (for the mentioned feature), i.e., 5.11 to 5.13, is narrower than ours, i.e., 5.5.1 to 5.13.